MariaDB query broken for multiple tags

The sql query is rejected for multiple tags :(

<details>

``` 
Caused by: java.sql.SQLException: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'INTERSECT (SELECT  itemid  FROM  tagitem  WHERE  tid  = '2gXqiQxhM7A-h3w7TW5EmQT' at line 1
```

</details>

MariaDB 10.3.22

--- a/modules/store/src/main/scala/docspell/store/queries/QItem.scala
+++ b/modules/store/src/main/scala/docspell/store/queries/QItem.scala
@@ -328,17 +328,14 @@ object QItem {
     val EC = REquipment.Columns
 
     // inclusive tags are AND-ed
-    val tagSelectsIncl = (q.tagsInclude
+    val tagSelectsIncl = q.tagsInclude
       .map(tid =>
         selectSimple(
           List(RTagItem.Columns.itemId),
           RTagItem.table,
           RTagItem.Columns.tagId.is(tid)
         )
-      ) ++ q.tagCategoryIncl.map(cat =>
-      TagItemName.itemsInCategory(NonEmptyList.of(cat))
-    ))
-      .map(f => sql"(" ++ f ++ sql") ")
+      ) ++ q.tagCategoryIncl.map(cat => TagItemName.itemsInCategory(NonEmptyList.of(cat)))
 
     // exclusive tags are OR-ed
     val tagSelectsExcl =

